City-of-Helsinki/drupal-helfi-platform-config
# UHF-1173: Matomo refactor

Moving Matomo & Matomo reports modules and config to `helfi_platform_config`.

How to test:

1. Checkout a fresh environment
2. Checkout this branch of the module: `composer require drupal/helfi_platform_config:dev-UHF-1173_matomo_refactor`
3. Build the site: `make new`
4. Enable the `HELfi Matomo configuration` feature
5. Clear cache, just to be sure
6. See the Matomo modules enabled and config in place
